tests/api.c

```diff
@@ -14840,6 +14840,45 @@ static void test_wc_PKCS7_EncodeEncryptedData (void)
 #endif
 } /* END test_wc_PKCS7_EncodeEncryptedData() */
 
+/*----------------------------------------------------------------------------*
+ | hash.h Tests
+ *----------------------------------------------------------------------------*/
+static void test_wc_HashInit(void) 
+{
+    /*enum for holding supported algorithms, #ifndef's restrict if disabled*/
+    enum wc_HashType enumArray[] = {
+    #ifndef NO_MD5
+            WC_HASH_TYPE_MD5,
+    #endif
+    #ifndef NO_SHA
+            WC_HASH_TYPE_SHA,
+    #endif
+    #ifndef NO_SHA224
+            WC_HASH_TYPE_SHA224,
+    #endif 
+    #ifndef NO_SHA256  
+            WC_HASH_TYPE_SHA256,      
+    #endif 
+    #ifndef NO_SHA384  
+            WC_HASH_TYPE_SHA384,      
+    #endif     
+    #ifndef NO_SHA512  
```

Comment:
There is not a NO_SHA512 define. Please use WOLFSSL_SHA512 instead. This is defined when SHA-512 is compiled into the library.

Reply:
thank you